### ISSUE TITLE
rnv@0.36.0-canary.9 is missing dist/core/constants.js

### ISSUE THREAD

**Issue by dotted** [renative-org/renative#838]
**Describe the bug**
Cannot `rnv new` due to `dist/core/constants.js` missing in `rnv` package

**To Reproduce**
1. `npm i -g rnv@0.36.0-canary.9`
2. Run `rnv new`
3. See error

**Expected behavior**
Not an error

**Screenshots or copy&paste**
```
➜  test rnv new
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '../constants'
Require stack:
- /home/bo/.nvm/versions/node/v16.14.2/lib/node_modules/rnv/dist/core/configManager/config.js
- /home/bo/.nvm/versions/node/v16.14.2/lib/node_modules/rnv/dist/cli/prompt.js
- /home/bo/.nvm/versions/node/v16.14.2/lib/node_modules/rnv/dist/core/common.js
- /home/bo/.nvm/versions/node/v16.14.2/lib/node_modules/rnv/dist/index.js
- /home/bo/.nvm/versions/node/v16.14.2/lib/node_modules/rnv/bin/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/bo/.nvm/versions/node/v16.14.2/lib/node_modules/rnv/dist/core/configManager/config.js:4:18)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/bo/.nvm/versions/node/v16.14.2/lib/node_modules/rnv/dist/core/configManager/config.js',
    '/home/bo/.nvm/versions/node/v16.14.2/lib/node_modules/rnv/dist/cli/prompt.js',
    '/home/bo/.nvm/versions/node/v16.14.2/lib/node_modules/rnv/dist/core/common.js',
    '/home/bo/.nvm/versions/node/v16.14.2/lib/node_modules/rnv/dist/index.js',
    '/home/bo/.nvm/versions/node/v16.14.2/lib/node_modules/rnv/bin/index.js'
  ]
}
```

**Desktop (please complete the following information):**

-   OS: Ubuntu 20.04 on WSL2
-   Node Version v16.14.2
-   RNV Version 0.36.0-canary.9

**Comment by pavjacko** [comment]
@dotted thx for reporting it. TS migration leftovers in publish scripts needed to be updated. should be fixed in latest `0.36.0-canary.12`

**Comment by dotted** [comment]
Confirmed, thanks!
